http/examples/http_async.rs

```diff
@@ -9,6 +9,7 @@ fn main() {
 
 	let server = ServerBuilder::new(io)
 		.cors(DomainsValidation::AllowOnly(vec![AccessControlAllowOrigin::Null]))
+		.threads(8)
```

Comment:
```suggestion
		.threads(4)
```
Maybe a saner default?

Reply:
Removed this altogether, pushed by mistake :)